### ISSUE TITLE
Turn off logs

### ISSUE THREAD

**Issue by pandres95** [AcalaNetwork/chopsticks#692]
Is there a way to disable logs from WASM runtime?

**Comment by xlc** [comment]
use env LOG_LEVEL=error

**Comment by pandres95** [comment]
Sorry, I meant not from the CLI, but using the `setup`, `setupWithServer` functions 😅

Even though, that's a pretty good solution for now.

**Comment by xlc** [comment]
Right. We can add a log level parameter to be able to configure it.
